### ISSUE TITLE
Exception with one core CPU

### ISSUE THREAD

**Issue by vlad4800** [dblock/oshi#70]
We have a Beaglebone that has only one cpu:
```
cat /proc/cpuinfo
processor       : 0
model name      : ARMv7 Processor rev 2 (v7l)
BogoMIPS        : 298.24
Features        : swp half thumb fastmult vfp edsp thumbee neon vfpv3 tls
CPU implementer : 0x41
CPU architecture: 7
CPU variant     : 0x3
CPU part        : 0xc08
CPU revision    : 2

Hardware        : Generic AM33XX (Flattened Device Tree)
Revision        : 0000
Serial          : 0000000000000000

```

Using the library to get CPU usage info throws an exception:
```
java.lang.IllegalArgumentException: Processor number (0) must be less than the number of CPUs: 0
        at oshi.software.os.linux.proc.CentralProcessor.<init>(CentralProcessor.java:107) ~[agents.jar:?]
        at oshi.software.os.linux.LinuxHardwareAbstractionLayer.getProcessors(LinuxHardwareAbstractionLayer.java:78) ~[agents.jar:?]

```

That's because in [CentralProcessor.java](https://github.com/dblock/oshi/blob/master/src/main/java/oshi/software/os/linux/proc/CentralProcessor.java) we have:
```
static {
		try {
			List<String> procCpu = FileUtil.readFile("/proc/cpuinfo");
			for (String cpu : procCpu) {
				if (cpu.startsWith("core id")) {
					numCPU++;
				}
			}
		} catch (IOException e) {
			System.err.println("Problem with: /proc/cpuinfo");
			System.err.println(e.getMessage());
		}
	}
```
But in this case "core id" does not exist.

Maybe you should default that to 1 core, somehow?

**Comment by dbwiddis** [comment]
Defaulting to 1 core is a good start.  Might also want to count lines starting with "processor" rather than "core id".

**Comment by dbwiddis** [comment]
Fixed in #76.
